app/scripts/controllers/footer.ctrl.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+angular.module('yourStlCourts').controller('FooterCtrl', function () {
+  var ctrl = this;
+
+  ctrl.copyRightYears = function(startingYear){
```

Comment:
The starting year doesn't really need to be a param. It's got to be hard-coded anyway, so might as well explicitly set it here in the controller.